portfolio/src/main/webapp/bandGen.js

```diff
@@ -0,0 +1,22 @@
+function getMood() {
+    let menu = document.getElementById("mood");
+    let mood = menu.options[menu.selectedIndex].text;
+    return mood;
+}
+
+function generateBand() {
+    let mood = getMood();
+    let band = addBands();
+    let bandContainer = document.getElementById("bandContainer");
+    let rand = Math.floor(Math.random() * Math.floor(2));
```

Comment:
You can just use 2 instead of `Math.floor(2)`, right?

Also, it's a good practice to extract "magic numbers" to a constant. In this case, I think `2` refers to the size of each band array, right? For someone else reading the code, it's better to see something like `Math.random() * NUMBER_OF_BANDS` rather than `Math.random() * 2` without any context, specially because the bands are created in another context (another function).